<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpPage</class>
 <widget class="QWidget" name="HelpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btn_home">
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/light/home.png</normaloff>:/icons/light/home.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="_ignore_horizontal_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="le_search_field">
         <property name="styleSheet">
          <string notr="true">QLineEdit#le_search_field
{
    border-top-left-radius: 10%;
    border-bottom-left-radius: 10%;
    border-right: none;
}</string>
         </property>
         <property name="-style-id" stdset="0">
          <string>browse-edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_search">
         <property name="styleSheet">
          <string notr="true">QToolButton#btn_search
{
    border-top-right-radius: 10%;
    border-bottom-right-radius: 10%;
    border-left: none;
}</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/light/search.png</normaloff>:/icons/light/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
         <property name="-style-id" stdset="0">
          <string>browse-btn</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="browser">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
